cockatrice/src/game/zones/view_zone_widget.cpp

```diff
@@ -43,20 +43,49 @@ ZoneViewWidget::ZoneViewWidget(Player *_player,
 
     // If the number is < 0, then it means that we can give the option to make the area sorted
     if (numberCards < 0) {
-        QGraphicsLinearLayout *hPilebox = new QGraphicsLinearLayout(Qt::Horizontal);
         QGraphicsLinearLayout *hFilterbox = new QGraphicsLinearLayout(Qt::Horizontal);
 
+        // groupBy label
+        groupByLabel.setFixedHeight(23);
+        groupByLabel.setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
```

Comment:
by giving the label a fixed width, does it work properly when the text in it changes with a retranslate?